src/components/Wizard/CreateVmWizard/CreateVmWizard.js

```diff
@@ -56,7 +57,7 @@ const getUpdatedValidatedState = (prevProps, prevState, props, state, extra) =>
 
   const validatedStepData = ALL_TAB_KEYS.reduce((resultStepData, tabKey) => {
     const { value, valid } = newState.stepData[tabKey];
-    resultStepData[tabKey] = validateTabData(tabKey, value, valid);
+    resultStepData[tabKey] = validateTabData(tabKey, value, valid, { [VIRTUAL_MACHINES_KEY]: extra.virtualMachines });
```

Comment:
why use some extra for that, can't we rather just pass the props for better convenience and to be more generic?

Reply:
Done!